doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -257,17 +257,31 @@ Connectivity Bridge
 nRF5340 Audio
 -------------
 
-* Added CAP initiator for the Bluetooth LE Audio unicast (CIS) client.
+* Added:
+
+  * CAP initiator for the Bluetooth LE Audio unicast (CIS) client.
+  * Support for any context type, not just media.
+  * Rejection of connection if unicast_client/broadcast_source tries to use an unsupported sample rate.
+  * Missing data to advertising packet after directed advertising has timed out.
+  * Debug prints of discovered endpoints.
+  * Support for multiple unicast_servers in unicast_client, regardless of location.
+  * Common Audio Profile (CAP) to unicast_client, including Coordinated Set Identification Profile (CSIP).
 
 * Removed:
 
   * The LE Audio controller for nRF5340 library.
     The only supported controller for LE Audio is :ref:`ug_ble_controller_softdevice`.
     This enables use of standard tools for building, configuring, and DFU.
+  * Source PAC records from unicast_server if only sink is supported.
 
 * Updated:
 
   * Low latency configuration to be used as default setting for the nRF5340 Audio application.
+  * ACL interval for service discovery to reduce set up time.
+  * Connection procedure so a central will not try to connect to the same device twice.
+  * Default settings to be lower latency end-to-end.
+  * Presentation delay calculation to be railed between min and max of the unicast_server.
```

Comment:
```suggestion
  * Presentation delay calculation so that it is railed between min and max of the :ref:`unicast server<nrf53_audio_unicast_server_app>`.
```